docs/docs/getting-started/installation/configuration.md

```diff
@@ -50,11 +50,6 @@ cluster and Redis.
   - `schema-registry` url to the Schema Registry
   - `commitInterval` the [Kafka Commit Interval](https://kafka.apache.org/documentation/#consumerconfigs_auto.commit.interval.ms) if you are using the included Helm chart
 
-- `redis`
-
-  - `hostname`
-  - `port`
-
 ### Security
```

Comment:
I think now it just needs to be documented then this is ready to go. Giving an approval of appreciation 🙌

Reply:
I will add documentation to this PR https://github.com/airyhq/airy/pull/2017 since then we also have a better way to actually deploy connectors and as discussed with @ljupcovangelski we might do a separate helm chart for Kafka Connect